Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionCocoa.mm

```diff
@@ -240,6 +241,12 @@
 
 #endif // PLATFORM(MAC)
 
+bool WebExtension::isAccessibleResourcePath(NSString *, NSURL *pageURL)
+{
+    // FIXME: Implement web accessible resources.
```

Comment:
Nit: Is there a bug number? Or do we not include those here.

Reply:
I'll file some bugs for these `FIXME`s.